lib/primitives/mtx.js

```diff
@@ -31,7 +33,7 @@ const {types} = rules;
  * @alias module:primitives.MTX
  * @extends TX
  * @property {Number} changeIndex
- * @property {CoinView} view
+ * @property {WalletCoinView} view
```

Comment:
In terms of abstractions, I am wondering if it makes sense for General MTX to have WalletCoinView, maybe change the name to something more appropriate ?

Reply:
Oh, good call. This comment is remnants of an older design. The general MTX will begin with a general CoinView object. I'll update the jsdoc.